custom/icds_reports/templates/icds_reports/icds_app/mobile/program-summary.directive.html

```diff
@@ -1,16 +1,38 @@
-{% load i18n %}
-<div cg-busy="[$ctrl.myPromise, $rootScope.locationPromise]">
-    <div class="row no-margin">
-      <h1 ng-bind="$ctrl.label"></h1>
-    </div>
-    <div class="row">
-        <div class="dot-menu">
-            <div class="dot-menu-item pointer" ng-repeat="step in $ctrl.steps">
-                <dot-link label="{$ step.label $}" route="{$ step.route $}"></dot-link>
+{% load staticfiles %}
+<div class="app-bg h-100vh overflow-hidden">
+    <div class="top-section p-3" style="padding-bottom: 25vh">
+        <div class="d-flex justify-content-between align-items-start mb-4">
+            <div class="d-flex align-items-start">
+                <img src="{% static "icds_reports/mobile/images/menu-icon.png" %}" class="mr-3">
+                <div>
+                    <p class="font-16 font-montserrat font-weight-600 text-white mb-0">
+                        Program Summary
+                    </p>
+                    <p class="font-14 font-montserrat font-weight-500 text-white">
+                        National | October 2019
```

Comment:
Just noticed this - is this supposed to be hard coded?

Reply:
"supposed to" is a bit ambiguous but it is known. i may be able to fix it, but we haven't actually gotten the month pickers built yet so the plan was to fix it when we integrate those.